test/cases/vmware.sh

```diff
@@ -124,6 +124,10 @@ version = "0.0.1"
 [[packages]]
 name = "bash"
 
+[[packages]]
+name = "ipa-client"
+version = "*"
```

Comment:
IIRC the version is not needed when we don't specify one

Reply:
I find it more explicit that way. Let me know if you have strong objections.

Reply:
Makes sense, no objection on my side